akka-stream-tests/src/test/java/akka/stream/io/OutputStreamSourceTest.java

```diff
@@ -29,11 +29,11 @@ public OutputStreamSourceTest() {
     }
 
     @ClassRule
-    public static AkkaJUnitActorSystemResource actorSystemResource = new AkkaJUnitActorSystemResource("OutputStreamSource",
+    public static AkkaJUnitActorSystemResource actorSystemResource = new AkkaJUnitActorSystemResource("OutputStreamSourceTest2",
             Utils.UnboundedMailboxConfig());
     @Test
     public void mustSendEventsViaOutputStream() throws Exception {
-        final FiniteDuration timeout = FiniteDuration.create(300, TimeUnit.MILLISECONDS);
+        final FiniteDuration timeout = FiniteDuration.create(3000, TimeUnit.MILLISECONDS);
```

Comment:
perhaps he just ment the syntax `3, TimeUnit.SECONDS` instead of `3000, TimeUnit.MILLISECONDS`?
yeah, that would read better